### DISCUSSION TITLE
萌新求助

### DISCUSSION THREAD

**Discussion by Winj1e** [nonebot/discussions#138] [问答]
![image](https://user-images.githubusercontent.com/105339518/167780765-ba22cd92-e1b7-47a5-a267-4b0f813a6d55.png)

这个问题怎么解决啊?

**Comment by yanyongyu** [comment]
loop的问题可以参考https://github.com/nonebot/discussions/discussions/13#discussioncomment-1159147

**Reply by Winj1e** [reply]
多谢解答，可是我还是不知道怎么解决

**Comment by MingxuanGame** [comment]
你可能是在 Windows 平台开启了冷重载 `fastapi_reload`

这会在 Windows 平台引发不可预期的行为

文档参考：[fastapi_reload](https://v2.nonebot.dev/docs/tutorial/choose-driver#fastapi_reload)

猴子补丁参考：[discussions#13 (comment)](https://github.com/nonebot/discussions/discussions/13#discussioncomment-1159147)

**Reply by Winj1e** [reply]
多谢解答，不过我看不懂这个文档

**Reply by MingxuanGame** [reply]
可以将冷重载关闭
修改配置参考：[配置](https://v2.nonebot.dev/docs/tutorial/configuration)

--------

也可以将猴子补丁写入到一个`.py`文件
在 `nonebot.run()` 执行之前导入

**Reply by Winj1e** [reply]
多谢大佬耐心解答，是这样的，这个机器人我上午还在用，后面我就去插件商店找新的插件，然后我加了几个插件后重新运行了一遍然后就出现了这个问题，这个机器人是我在b站看视频搞出来的，其实我什么都不懂，我还是放弃吧，不好意思浪费大家的时间了

**Reply by yanyongyu** [reply]
你新加的某个插件用到了htmlrender生成图片，但是playwright和uvicorn在windows上存在一定的冲突

**Reply by Winj1e** [reply]
那是不是说我把那个插件删了是不是就能好了

**Reply by yanyongyu** [reply]
是的

**Reply by Winj1e** [reply]
那么我要在哪卸载新装的插件呢？我找不到他们被装在哪了

**Reply by MingxuanGame** [reply]
如果你是使用脚手架安装的

可以用 `nb plugin remove 插件名`

**Reply by Winj1e** [reply]
好的，谢谢大佬们解答
